interface/super/edit_list.php

```diff
@@ -818,147 +810,162 @@ function mysubmit() {
 </head>
 
 <body class="body_top">
+<nav class="navbar navbar-default navbar-fixed-top">
+<div class="container-fluid">
```

Comment:
another likely dumb question. Does it ever make sense to keep the navbar in a container-fluid div, but then keep the page content below it in a separate container div?